### PR TITLE
Add initial GMX V1 support

### PR DESCRIPTION
**Pull request by yabirgb**
This PR adds decoders for swaps in GMX and increase and decrease of positions

Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/tests/unit/decoders/test_gmx.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+            tx_hash=evmhash,
+            sequence_index=20,
+            timestamp=timestamp,
+            location=Location.ARBITRUM_ONE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(FVal(fee_amount)),
+            location_label=user_address,
+            notes=f'Spend {fee_amount} ETH as GMX fee',
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/decoders/test_gmx.py:183]: fee is always in ETH?

**Review by LefterisJP — COMMENTED** [review]
Some questions/comments.

Also should it not have a changelog entry?

File: rotkehlchen/tests/unit/decoders/test_gmx.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/tests/unit/decoders/test_gmx.py:183]: No, it depends on the pair queried. This was for longing ETH and the fee is in ETH

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**